patches/chrome-browser-android-omnibox-autocomplete_controller_android.cc.patch

```diff
@@ -1,16 +1,16 @@
 diff --git a/chrome/browser/android/omnibox/autocomplete_controller_android.cc b/chrome/browser/android/omnibox/autocomplete_controller_android.cc
-index 08685131fd1de67feb5fc28acc33e080fd51b64c..5f3d675b30845c7cbc1d1a58b73380c568d92099 100644
+index d2f549913162b2def9316840e61ebd3b8e9c86f4..a9fc95c23d5a0110ac2a4596c34e865b6ba4847f 100644
 --- a/chrome/browser/android/omnibox/autocomplete_controller_android.cc
 +++ b/chrome/browser/android/omnibox/autocomplete_controller_android.cc
-@@ -16,6 +16,7 @@
+@@ -17,6 +17,7 @@
  #include "base/strings/utf_string_conversions.h"
  #include "base/time/time.h"
  #include "base/timer/timer.h"
 +#include "brave/components/omnibox/browser/brave_autocomplete_controller.h"
```

Comment:
Do we need to patch android file?

Reply:
We do eventually want to combine the android work into brave-core, but we're not there yet and for normal dev we haven't kept up with Android changes.  
That being said this patch isn't introduced here, but the leading plus makes it look like it is.   The added lines will have 2 `+` or `-`.